erica_app/erica/request_processing/erica_input.py

```diff
@@ -89,48 +89,116 @@ class FormDataEst(BaseModel):
     stmind_gem_haushalt_count: Optional[int]
     stmind_gem_haushalt_entries: Optional[List[str]]
 
+    @root_validator(pre=True)
+    def disability_merkzeichen_and_pflegegrad_flags_must_be_boolean(cls, values):
+        merkzeichen_names = [
+            'person_a_has_pflegegrad',
+            'person_a_has_merkzeichen_bl',
+            'person_a_has_merkzeichen_tbl',
+            'person_a_has_merkzeichen_h',
+            'person_a_has_merkzeichen_g',
+            'person_a_has_merkzeichen_ag',
+            'person_b_has_pflegegrad',
+            'person_b_has_merkzeichen_bl',
+            'person_b_has_merkzeichen_tbl',
+            'person_b_has_merkzeichen_h',
+            'person_b_has_merkzeichen_g',
+            'person_b_has_merkzeichen_ag',
+        ]
+        for merkzeichen_name in merkzeichen_names:
+            merkzeichen_value = values.get(merkzeichen_name)
+            if merkzeichen_value is not None and not isinstance(merkzeichen_value, bool):
+                raise ValueError(f"{merkzeichen_name} is not a boolean. {merkzeichen_value} was given")
+        return values
+
+    @root_validator(pre=True)
+    def set_person_a_merkzeichen_bl_tbl_h_pflegegrad_set(cls, values):
+        merkzeichen_values = [
+           values.get('person_a_has_pflegegrad'),
+           values.get('person_a_has_merkzeichen_bl'),
+           values.get('person_a_has_merkzeichen_tbl'),
+           values.get('person_a_has_merkzeichen_h')
+        ]
+        if any([merkzeichen_value is True for merkzeichen_value in merkzeichen_values]):
```

Comment:
Isn't this the same this as writing the following?
```suggestion
        merkzeichen_values = [
           values.get('person_a_has_pflegegrad'),
           values.get('person_a_has_merkzeichen_bl'),
           values.get('person_a_has_merkzeichen_tbl'),
           values.get('person_a_has_merkzeichen_h')
        ]
        if any(merkzeichen_values]):
```

Reply:
Yep 🙈